CHANGELOG.md

```diff
@@ -22,6 +22,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Added: FAQ Entry about non-Superheated rooms
 
+#### Chozo Ruins
```

Comment:
Should be under the section:

```
#### Logic Database
```

(create if there is none)

Reply:
something like this?